Point everything at the right hosts and principals for the test realm
for right now.  At some point, we need a better way of dealing with
test vs. production realms.

Index: kadmin-remctl/passwd_change.c
===================================================================
--- kadmin-remctl.orig/passwd_change.c	2007-05-31 16:35:21.000000000 -0700
+++ kadmin-remctl/passwd_change.c	2007-05-31 16:43:00.000000000 -0700
@@ -28,10 +28,10 @@
 #define PASSWD_FILE "/afs/ir/service/etc/passwd.all"
 
 /* The principal name used for password changing. */
-#define PRINCIPAL "service/password-change@stanford.edu"
+#define PRINCIPAL "service/password-change@test-k5.stanford.edu"
 
 /* The host and port to which the remctl connection should be made. */
-#define HOST "lsdb.stanford.edu"
+#define HOST "kerberos-dev.stanford.edu"
 #define PORT 4443
 
 /* The memory cache used for the password change authentication. */
